include/picongpu/fields/absorber/pml/Pml.kernel

```diff
@@ -237,6 +237,11 @@ namespace picongpu
                 class UpdateEFunctor
                 {
                 public:
+                    //! Stencil requirements for lower margins
+                    using LowerMargin = typename traits::GetLowerMargin<T_CurlB>::type;
```

Comment:
Could it be that using the margins of `T_CurlB` is larger than we need for the PMLs?
Maybe the requirements for upper or lower margins can be reduced?
This question is coming up because you question if we can provide a unified class which provides upper and lower margin within one unified class based on curl.

Reply:
I partly clarified my point in another thread at the same time. Briefly: with the current implementation it is not possible. And even if PML did same as exponential daming, so point-wise uptates, the functor also handles the non-absorbing area. So even then the requirements would remain the same.